api/models/Post.js

```diff
@@ -176,7 +176,8 @@ module.exports = bookshelf.Model.extend(Object.assign({
   },
 
   isPublic: function () {
-    return this.get('visibility') === Post.Visibility.PUBLIC_READABLE
+    // return this.get('visibility') === Post.Visibility.PUBLIC_READABLE
```

Comment:
lets decide which to use and remove comment if we want to go with the boolean

Reply:
👍